doc/source/reference/random/brng/index.rst

```diff
@@ -0,0 +1,39 @@
+.. _bit_generator:
+
+Bit Generators
+--------------
+
+.. currentmodule:: numpy.random
+
+The random values produced by :class:`~Generator` 
+orignate in a BitGenerator.  The BitGenerators do not directly provide
+random numbers and only contains methods used for seeding, getting or
+setting the state, jumping or advancing the state, and for accessing 
+low-level wrappers for consumption by code that can efficiently 
+access the functions provided, e.g., `numba <https://numba.pydata.org>`_.
+
+Stable RNGs
+===========
+
+.. toctree::
+   :maxdepth: 1
+
+   DSFMT <dsfmt>
+   MT19937 <mt19937>
+   Philox <philox>
+   ThreeFry <threefry>
+   XoroShiro128+ <xoroshiro128>
+   Xorshift1024*φ <xorshift1024>
+   Xoshiro256** <xoshiro256starstar>
```

Comment:
I've been wondering if xoshiro256starstar should be renamed to xoshiro256.  Sameo for the 512 version. Thedocs canmake it clear which version this is (there is a + and a ** version).

Reply:
I have made a PR that makes ths change.  @mattip LMK what you think.  These seem a bit verbose to me.